tensorflow/core/kernels/fft_ops.cc

```diff
@@ -135,9 +134,52 @@ class FFTCPU : public FFTBase {
         output.device(device) =
             full_fft.slice(startIndices, output.dimensions());
       } else {
-        // TODO: reconstruct the full fft and take the inverse.
-        ctx->CtxFailureWithWarning(
-            errors::Unimplemented("IRFFT is not implemented as a CPU kernel"));
+        // Reconstruct the full fft and take the inverse.
+        auto input = ((Tensor)in).flat_inner_dims<complex64, FFTRank + 1>();
+        auto output = out->flat_inner_dims<float, FFTRank + 1>();
+
+        Eigen::DSizes<Eigen::DenseIndex, FFTRank + 1> startIndices;
+        auto sizes = input.dimensions();
+
+        // Calculate the shape of full-fft temporary tensor
```

Comment:
style nit: please end sentences in comments with periods (here and below)
https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar